src/shared/acl-util.c

```diff
@@ -232,7 +232,7 @@ int parse_acl(char *text, acl_t *acl_access, acl_t *acl_default, bool want_mask)
 
         split = strv_split(text, ",");
         if (!split)
-                return log_oom();
+                return -ENOMEM;
```

Comment:
It's a library function, and thus shouldn't log on its own.

Generally in systemd we follow the rule that:

a) either a function logs all errors or none (in which case the caller should log)

b) "library" functions (which are the ones in src/shared/ and src/basic/) should never log errors, but leave that to the callers.

Before this change parse_acl() logged some errors and others it didn't log (what#s even worse: it logged some OOM errors but not other OOM errors!). Also, the function is a library function, hence shouldn't log at all. I hence removed the log_oom(), and replaced it with a simple -ENOMEM return.

Also, see tmpfiles.c which is the only user of the call currently, and which logs on its own.